src/FastSerialization/SegmentedMemoryStreamReader.cs

```diff
@@ -174,6 +204,11 @@ public void Dispose()
         /// Dispose pattern
         /// </summary>
         protected virtual void Dispose(bool disposing) { }
+
+        /// <summary>
+        /// Returns the SerializationConfiguration for this stream reader.
+        /// </summary>
+        public SerializationConfiguration SerializationConfiguration { get; private set; }
```

Comment:
If not needed, let's keep this internal.  I'm not sure how much of this is going to live forever vs. disappear relatively quickly once we have backwards compat in-place.